### DISCUSSION TITLE
Does Lume support Deno 2 ?

### DISCUSSION THREAD

**Discussion by hanguokai** [lumeland/lume#684] [Q&A] [answered]
Recently, Deno 2 was released. I haven't test it. Just want to know which Deno version can be used for Lume?

**Comment by marvinhagemeister** [comment, marked as answer]
Yes, lume supports Deno 2. We're using lume for `docs.deno.com` ourselves here at Deno together with Deno 2.

**Reply by hanguokai** [reply]
Thanks
